frontend/src/App.svelte

```diff
@@ -109,15 +161,12 @@
         return ret;
       }
     })();
-
```

Comment:
whats up with all the deleted lines? please align the formatting with what is already established.

Reply:
Made some changes, let me know if that's what you meant.